ROMFS/px4fmu_common/mixers/zmr250.main.mix

```diff
@@ -0,0 +1,10 @@
+# R: <geometry> <roll scale> <pitch scale> <yaw scale> <deadband>
+R: 4x 7654 10000 10000 0
```

Comment:
Ok, never mind my previous post then.
Mixer config is the best way to accommodate custom geometry. ZMR 250 is asymmetric and mixer compensates this.
For ZMR250 mixing for cleanflight/betaflight please refer to https://oscarliang.com/custom-motor-output-mix-quadcopter/